### ISSUE TITLE
Not available for #SailfishOS anymore..

### ISSUE THREAD

**Issue by jjaone** [khertan/ownNotes#11]
Why? Do you have any plans to still support #SailfisOS ? This has been a very useful app, which hopefully someone will contiinue to develop for #Jolla.. Are there some specific issues that need to be resolved or is this because of update 9 and Qt 5.2 that the app does not work in #SailfishOS anymore?

**Comment by llewelld** [comment]
ownNotes has been a core app for me on SailfishOS, and although I understand you're not supporting it on the platform, I'm sure there are others like me who find it useful. Unfortunately ownCloud removed support for WebDAV locks, and since then it didn't work for me.

I've finally had a chance to look at your source to remove the locking code, and with this minor change it seems to be working again. I'll create a pull request in case you're interested to incorporate the changes into yours.

Are you happy for me to make a package available on OpenRepos again (unless you prefer to do it yourself)?

**Comment by khertan** [comment]
Of course, you are free to make the package yourself for OpenRepos, upload it and take the leadership on it.

**Comment by brvier** [comment]
Thanks for the pull request

**Comment by llewelld** [comment]
Thank you for being so accommodating, and for merging the pull request.

'take the leadership' may be a bit too strong, but I'll certainly make the package available for others to use as it's such a useful app.

**Comment by jean-clovis** [comment]
yes, some of us were not even expecting some news regarding this app, thanks both of you for raising it back from the dead :D

**Comment by llewelld** [comment]
Sorry, I must also ask: is there a way of generating the HTML changelogs to display in the 'About' page? I couldn't see this documented anywhere, but maintaining this in the app is a nice idea.

**Comment by brvier** [comment]
There is a script i used for this /Sailfish/ownNotes/release.py, it s use git tag to generate changelog based on some basic formatting in changelog. I didn t use it since years, so didn t remember how it works exactly :p

**Comment by llewelld** [comment]
Thank you, that's perfect. I'll give it a go :)

**Comment by llewelld** [comment]
I managed to get to grips with release.py (after generating a few releases with peculiar names while I figured out the command-line parameters!).

The new version is up on OpenRepos for anyone interested:

https://openrepos.net/content/flypig/ownnotes
http://www.flypig.co.uk/?to=ownnotes

I'll be doing my best to maintain this version, but am on holiday right now so will suddenly have much less time when I get back to work :( Any contributions from others would be very welcome :)

**Comment by llewelld** [comment]
Thanks again, @khertan, for your support.
